product_auction/models/product_template.py

```diff
@@ -9,6 +9,9 @@ class ProductTemplate(models.Model):
 
     case_number = fields.Char()
     product_model_id = fields.Many2one("product.model", "Model")
+    product_model_category_id = fields.Many2one(
+        related="product_model_id.category_id", readonly=False, store=True
```

Comment:
```suggestion
        related="product_model_id.category_id", store=True
```
Any reason to assign `readonly=False`?

Reply:
Yes. We make automatically fill with relevant one. Just in case, if the user want to change model category for any reason, I want to let them change.May be rare condition.